lib/src/component_declaration/component_type_checking.dart

```diff
@@ -48,14 +49,17 @@ const String _componentTypeMetaKey = '_componentTypeMeta';
 /// the component type of the specified [factory].
 ///
 /// This meta is retrievable via [getComponentTypeMeta].
-// ignore: deprecated_member_use
-void setComponentTypeMeta(ReactDartComponentFactoryProxy factory, {
-    bool isWrapper,
-    // ignore: deprecated_member_use
-    ReactDartComponentFactoryProxy parentType
+void setComponentTypeMeta(ReactComponentFactoryProxy factory, {
+  @required ReactComponentFactoryProxy parentType,
```

Comment:
Is it gucci to make a previously optional parameter required without a major release?

Reply:
waaaiit 🤦‍♂ scratch that